src/Windows/GeneratePackage/Azure.DCAP.Windows.nuspec

```diff
@@ -7,7 +7,7 @@
         <authors>Microsoft</authors>
         <owners>Microsoft</owners>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
-        <releaseNotes>Use the Cache-Control header to determine how long collateral should be cached for. Cleanup of error handling logs in Windows</releaseNotes>
+        <releaseNotes>Enable v3 for icx and coffelake and Use the Cache-Control header to determine how long collateral should be cached for. Cleanup of error handling logs in Windows</releaseNotes>
```

Comment:
I would just fill this in with what has changed. History will be available on the previous versions